IdokladSdk.IntegrationTests/Tests/Clients/ProformaInvoice/ProformaInvoiceTests.Async.cs

```diff
@@ -132,6 +133,37 @@ public async Task DeleteAsync_SuccessfullyDeleted()
             Assert.IsTrue(data);
         }
 
+        [Test]
+        public async Task AccountMultipleProformasAsync_SuccessfullyAccounted()
+        {
+            // Arrange
+            var proformaModel = (await _proformaInvoiceClient.DefaultAsync()).AssertResult();
```

Comment:
Radši bych to vytáhnul do nějaké private metody, pak to duplikuješ v testu ne-async metody.